### ISSUE TITLE
Figure out random seeds

### ISSUE THREAD

**Issue by blindjesse** [csp-inc/lumonitor#22]
I seem to be getting the same dataset out of the dataloader on every epoch.

**Comment by blindjesse** [comment]
I got things randomized for the training set by defining worker_init_fn and calling `np.random.seed()` at the beginning of each epoch. Still need to figure out how to get the exact same tiles and data for the training set :exploding_head:

**Comment by blindjesse** [comment]
See https://github.com/csp-inc/lumonitor/blob/dev/src/model/test_dataset.py. Basically if you set `shuffle=True` in call to dataloader it will shuffle files on each epoch. Defining `worker_init_fn` for dataset also give each file different windows BUT they are the same on each epoch. Nothing else I did (setting random seed before each epoch, for instance) seemed to have any effect.
